main.go

```diff
@@ -205,13 +205,13 @@ func updates(r *moduleReference, out chan outputUpdates) error {
 		}
 	}
 	switch {
-	case r.Version == nil && currentVersion == nil:
+	case *r.Version == "" && currentVersion == nil:
```

Comment:
not immediately mergeable because r.Version might be nil -- this would panic then. I can give fixing this a go once I find some time.

Reply:
Feel free to adapt or modify the change. Looking forward to your response.